zebrad/tests/acceptance.rs

```diff
@@ -16,11 +16,8 @@ pub fn tempdir(create_config: bool) -> Result<(PathBuf, impl Drop)> {
     let dir = TempDir::new("zebrad_tests")?;
 
     if create_config {
-        let cache_dir = dir.path().join("state");
-        fs::create_dir(&cache_dir)?;
-
         let mut config = ZebradConfig::default();
-        config.state.cache_dir = cache_dir;
+        config.state.ephemeral = true;
```

Comment:
Since we don't set a cache_dir here, the ephemeral database uses Zebra's default cache dir in all the parallel acceptance tests, causing conflicts.

If we want sled to choose a unique cache_dir, and use a memory cache on Linux, we must stop calling `sled::Config::path` when ephemeral is true. Otherwise, we should supply a unique `cache_dir` in the test config.